examples/plot_kernel_ridge_regression.py

```diff
@@ -136,9 +137,9 @@
         test_time.append(time.time() - t0)
 
     plt.plot(sizes, train_time, 'o-', color="r" if name == "SVR" else "g",
-             label="%s (train)" % name)
+             markeredgecolor=(0, 0, 0), label="%s (train)" % name)
```

Comment:
I am not convinced this is needed here and a few lines below [here](https://github.com/scikit-learn/scikit-learn/pull/8413/files#diff-bf53cf2bfe266309ed0c4bddd81a2bcbR142). Because there is a line connecting the dots I feel the dev plot is fine:
dev:
![](http://scikit-learn.org/dev/_images/sphx_glr_plot_kernel_ridge_regression_002.png)

For reference the plot from your PR:
![](https://8992-843222-gh.circle-artifacts.com/0/home/ubuntu/scikit-learn/doc/_build/html/stable/_images/sphx_glr_plot_kernel_ridge_regression_002.png)